lib/asana/resources/task.rb

```diff
@@ -126,17 +129,17 @@ def find_by_id(client, id, options: {})
         # Returns the compact task records for all tasks within the given project,
         # ordered by their priority within the project.
         #
-        # projectId - [Id] The project in which to search for tasks.
+        # project - [Gid] The project in which to search for tasks.
         # per_page - [Integer] the number of records to fetch per page.
         # options - [Hash] the request I/O options.
-        def find_by_project(client, projectId: required("projectId"), per_page: 20, options: {})
+        def find_by_project(client, project: required("project"), per_page: 20, options: {})
```

Comment:
Another parameter name change that we can't really mitigate; this one is almost assuredly going to hit people because I expect this to be used quite a bit 😢 

Guess we have another case for hand-patching.